README.md

```diff
@@ -9,15 +9,13 @@ paradigm and AssertJ style assertion) or by providing query files with expected
 
 ## Prerequisites
 
-You will need the following software to be installed on the machine running the
-framework:
+To use Tempto you need a Java 1.8 runtime.
 
-* Java >= 1.8
-* Python >= 2.6 (if you use the custom launcher that comes with the framework)
+There rest of dependencies depends of set of features you are using
 
-Currently we only support HDFS as a datastore. That means that on your Hadoop
-cluster you'll need the following:
+### HDFS
 
+For automatic provisioning HDFS based tables you need:
 * Running Hadoop cluster with [WebHDFS](http://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-hdfs/WebHDFS.html)
 * We suggest that cluster support [XAttr metadata](https://hadoop.apache.org/docs/r2.6.0/hadoop-project-dist/hadoop-hdfs/ExtededAttributes.html). 
```

Comment:
this link doesn't seem to be working anymore
https://hadoop.apache.org/docs/r2.6.0/hadoop-project-dist/hadoop-hdfs/ExtededAttributes.html